fe1-web/parts/lao/socialMedia/SocialProfile.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+import { Text, View } from 'react-native';
+
+const SocialProfile = () => (
+  <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
+    <Text>My Profile</Text>
```

Comment:
I think it's better to use the string you created in STRINGS.

Reply:
This is just an example and will be deleted for sure after finishing this page, but yes, I can change it.